### PR TITLE
Add toggle to use 24h time

### PR DESCRIPTION
**Pull request by zusorio**
This adds a toggle to use 24-hour time to the timezone selector.
image
It also
- Uses the formatTime helper in more places
- Modifies that helper to accept an object instead of several parameters
- Swaps the timezone selector to use bootstrap form elements
- Swaps the CET timezone from Europe/Paris to Europe/Berlin

### CODE REVIEW

**Comment by slmnio** [comment]
I like the concept, and I think it works in the modal well. It doesn't seem to flow amazingly in the events/match page as it pushes down the rest of the content.
image

File: website/src/utils/content-utils.js [hunk 1]
```diff
@@ -329,15 +329,21 @@ export function getAbbrev(timezone, time) {
...
+    const timezone = getTimezone(tz);
     const time = spacetime(timeString).goto(timezone);
     const abbrev = getAbbrev(timezone, time);
-    return time.format((format || "{day-short} {date-ordinal} {month-short} {year} {time} {tz}").replace("{tz}", abbrev));
+    return time.format(
+        (format || "")
+            .replace("{tz}", abbrev)
+            .replace("{time}", use24HourTime ? "{time-24}" : "{time}")
+            .replace("{year-short-prev-only}", time.year() === spacetime.now().year() ? "" : "{year-short}")
+    );
```
Comment (slmnio) [inline website/src/utils/content-utils.js:346]: Big fan of this. Does format need the `|| ""`? It should fall back to the default

Comment (slmnio) [reply website/src/utils/content-utils.js:346]: Also is it worth adding a `.trim()` at the end of the chain? The HTML will probably be fine but the format `"{date} {month-short} {year-short-prev-only}"` will end with an empty space

Comment (zusorio) [reply website/src/utils/content-utils.js:346]: You're right the  `|| ""` can be removed. 
The time i used `{year-short-prev-only}` I did add a `.trim()` but it should probably be built in.

**Comment by zusorio** [comment]
> It doesn't seem to flow amazingly in the events/match page as it pushes down the rest of the content.
I can add a prop to the component to display the toggle inline or just hide it completely on those pages

**Review by slmnio — APPROVED** [review]

**Pull request merged**